### PR TITLE
Rewrites to Developing Plugins topic

### PR DESCRIPTION
**Pull request by mdlinville**
See https://github.com/docker/docker.github.io/pull/263 where this work originated.

Clarified and tested the steps to create a plugin manually

### CODE REVIEW

**Comment by mstanleyjones** [comment]
@thaJeztah PTAL, fixed the typos.

File: docs/extend/index.md [hunk 1]
```diff
@@ -74,6 +78,8 @@ started without error.
     ```
 
 3.  Create a volume using the plugin.
+    This example mounts the `/remmote` directory on host `1.2.3.4` into a
```
Comment (thaJeztah) [inline docs/extend/index.md]: s/remmote/remote/

File: docs/extend/index.md [hunk 2]
```diff
@@ -74,6 +78,8 @@ started without error.
...
+    volume nammed `sshvolume`. This volume can now be mounted into containers.
```
Comment (thaJeztah) [inline docs/extend/index.md]: s/nammed/named/

**Review by thaJeztah — CHANGES_REQUESTED** [review]
hm, did you forget to stage/push the typo fixes? LGTM otherwise

File: docs/extend/index.md (hunk 1 above)
Comment (mstanleyjones) [inline docs/extend/index.md]: No, I forgot to amend!

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**